### PR TITLE
feat(legacy/tools): elevate karma

### PR DESCRIPTION
**Pull request by InteractiveTimmy**
# Description

The scope of the changes in this pull request are to elevate all karma-related tooling into the existing `@webex/legacy-tools` package for consumption within this project.

Screenshot 2023-04-05 at 8 15 34 PM

### CODE REVIEW

File: packages/@webex/internal-plugin-feature/esbuild.config.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const esbuild = require('esbuild');
```
Comment (arun3528) [inline packages/@webex/internal-plugin-feature/esbuild.config.js:1]: why esbuild here ?

File: packages/legacy/tools/src/models/package/package.ts [hunk 2]
```diff
@@ -116,12 +116,33 @@ class Package {
 
     return Promise.all([unitTestFileCollector, integrationTestFileCollector])
       .then(async ([unitFiles, integrationFiles]) => {
-        if (unitFiles.length > 0) {
-          await Jest.test({ files: unitFiles });
+        if (config.runner === 'jest') {
```
Comment (arun3528) [inline packages/legacy/tools/src/models/package/package.ts:119]: nice to see all together

File: packages/legacy/tools/test/integration/karma/karma.test.js [hunk 3]
```diff
@@ -0,0 +1,251 @@
+const KarmaRunner = require('karma');
+
+const { Karma } = require('@webex/legacy-tools');
+
```
Comment (arun3528) [inline packages/legacy/tools/test/integration/karma/karma.test.js:4]: looks clean

**Review by arun3528 — COMMENTED** [review]
looks good to me 

do we need the esbuild config , looks like needs some cleanup

File: packages/@webex/internal-plugin-feature/esbuild.config.js (hunk 1 above)
Comment (InteractiveTimmy) [reply packages/@webex/internal-plugin-feature/esbuild.config.js]: Removed this, was a file i missed when duplicating the setup of `internal-plugin-flag` to test within another plugin.

**Review by arun3528 — APPROVED** [review]

**Pull request merged**